airbyte-integrations/connectors/source-survey-sparrow/source_survey_sparrow/spec.yaml

```diff
@@ -18,3 +18,27 @@ connectionSpecification:
     survey_id:
       type: array
       description: A List of your survey ids for survey-specific stream
+    is_eu_based:
+      title: Base URL 
+      type: object
```

Comment:
adding `order` key is required now, do you mind to add it to all properties here?

Reply:
I've added docs, tests, and order key. :-) Feel free to comment if I'm still missing anything